portal.Dockerfile

```diff
@@ -1,22 +1,23 @@
 ARG registry=docker.osdc.io
+ARG NPM_REGISTRY="https://registry.npmjs.org/"
```

Comment:
In the build notes I had captured
```Node v8, npm v5
nvm install 8
nvm use 8
npm i -g npm@5
```
I am surprised to see node:13 and no mention of npm.... but if it works it works.

Reply:
I think the docker image we are using have npm 6.14.4

```
❯ docker run --rm -it -v "$PWD":/app -w /app docker.osdc.io/node:13 sh
# bash
root@4c8db8a850ed:/app# npm --version
6.14.4
```